node-repository/src/main/java/com/yahoo/vespa/hosted/provision/node/Agent.java

```diff
@@ -17,5 +17,6 @@
     FailedExpirer,
     InactiveExpirer,
     ProvisionedExpirer,
-    ReservationExpirer
+    ReservationExpirer,
+    DynamicProvisioningMaintainer
```

Comment:
Add to https://github.com/vespa-engine/vespa/blob/ddd5141d112d44f881da17e83d26ac18ad35c3c1/controller-api/src/main/java/com/yahoo/vespa/hosted/controller/api/integration/noderepository/NodeHistory.java#L35

Reply:
Thanks 

https://github.com/vespa-engine/vespa/pull/12523